src/patternfly/components/Switch/examples/Switch.md

```diff
@@ -23,6 +23,24 @@ cssPrefix: pf-c-switch
 {{/switch}}
 ```
 
+### Reverse (Toggle on right)
```

Comment:
Small nit, we use sentence case pretty much everywhere. 

```suggestion
### Reverse (toggle on right)
```

FYI I did a search to see how we're handling this in other examples and looks like it's lower case.